lib/services/devices/registrationUtils.js

```diff
@@ -317,15 +314,18 @@ function sendUnregistrationsNgsiLD(deviceData, callback) {
         options.url = 'http://' + deviceData.cbHost + '/ngsi-ld/v1/csourceRegistrations/' + deviceData.registrationId;
     }
     if (deviceData.registrationId) {
-        logger.debug(context, 'Sending device unregistrations to Context Broker at [%s]', options.url);
+        logger.debug(context, 'Sending device LD unregistrations to Context Broker at [%s]', options.url);
         logger.debug(context, 'Using the following request:\n\n%s\n\n', JSON.stringify(options, null, 4));
 
-        return deviceService.executeWithSecurity(
+        return utils.executeWithSecurity(
             options,
             deviceData,
-            createRegistrationHandlerNgsi2(true, deviceData, callback));
+            createRegistrationHandlerNgsi2(true, deviceData, callback)
+        );
     }
 
+    //console.error(JSON.stringify(options.json, null, 4));
```

Comment:
Leftover comment?

Reply:
Yes, but - the multiple instances of `console.error`  (commented out) are being used to help generate the responses for the tests.

When the final PR is processed I'll remove them all.

Reply:
Noted in the accumulative PR - https://github.com/telefonicaid/iotagent-node-lib/pull/849#issuecomment-594666173

NTC with regards this PR